sbom/spdx/encoder.go

```diff
@@ -0,0 +1,377 @@
+package spdx
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	spdxjson "github.com/spdx/tools-golang/json"
+	"io"
+	"slices"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/sbom"
+
+	"github.com/spdx/tools-golang/spdx/common"
+	v2common "github.com/spdx/tools-golang/spdx/v2/common"
+	"github.com/spdx/tools-golang/spdx/v2/v2_3"
+)
+
+type Version string
+
+const (
+	V2_3 Version = "v2.3"
+)
+
+type Format string
+
+const JSON Format = "json"
+
+type Creator struct {
```

Comment:
Is this required? There is already https://github.com/spdx/tools-golang/blob/v0.5.5/spdx/v2/common/creation_info.go

Reply:
I was thinking decoupling the spdx version with this information might be helpful. For example, the package you linked is specifically for spdx v2. I don't have a strong preference though.

Reply:
Sounds fair